CONTRIBUTING.md

```diff
@@ -46,7 +48,7 @@ All of the currently supported parity constraints (subclasses of `Moment`) are b
 
 #### Post-processing algorithms
 
-The constructors of post-processing algorithms require an already trained predictor as an argument. As an alternative, it is possible to provide an estimator, and the estimator is then fitted on the data at the beginning of the execution of `fit`. For post-processing algorithms, the `constraints` argument is provided as a string.
+The constructors of post-processing algorithms require an already trained predictor as an argument or, alternatively, an estimator that is subsequently fitted on the data when executing `fit`. For post-processing algorithms, the `constraints` argument is provided as a string.
```

Comment:
How about "algorithms require either a trained predictor or an estimator (which is trained on the data by executing :code:`fit`" ?

Reply:
I took your suggestion with a small change ("when" isntead of "by")